website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-choosing-a-dbt-version.md

```diff
@@ -22,16 +22,39 @@ Each job in dbt Cloud can be configured to inherit parameters from the environme
 
-Currently, you can ask dbt jobs to run versions of dbt Core prior to 0.14.x. Be aware that these versions may become unavailable in a future release. And because dbt Cloud leverages components of dbt Core, users running outdated or unsupported versions of Core may find certain features are buggy, disabled, or missing.
+| dbt Version | Release Date | Deprecation Date |
+| ------------- | -------------- | ------------- | 
+| v0.X.X (major version 0 ) | Various Dates | June 30, 2022 |
+| v1.0.0 | December 2021 | December 2022 |
+
+
+#### What will actually happen on the deprecation date?
+- On July 1, 2022, we will only run jobs that are on dbt v1.0 or later. Customers must upgrade their projects to be compatible with dbt v1.0 or later.
+- 1 year post a minor version release of v1.X, we will try to run our customers' projects on the latest release of dbt if they have not already upgraded their projects themselves. In a post dbt v1.0 world, there won't be breaking changes between minor versions of dbt, so we might be reasonably successful at upgrading our customers' versions automatically. However, our strong preference is for customers to try to manage the upgrade process themselves which is a more cautious way to prevent failures to their production pipelines. 
+
+We will give customers consistent communication that they're hitting the end of their supported window, so they can plan accordingly. 
+
+#### What should you be doing today?
+There are years of development and a handful of breaking changes between two distant versions of dbt (e.g. 0.14 --> 0.20). There are far fewer between two subsequent versions of dbt, which is why upgrading regularly is important. Getting your project running on the latest release of dbt today will make the jump to v1.0 much easier when it's released.
+
+Additionally upgrading to more recent versions of dbt Core will enable better performance and more features in dbt Cloud. Below is a compatability matrix between dbt versions and dbt Cloud features. Hopefully this provides more motivation to always update your environments and jobs to run the latest version of dbt.
+
+
+
+| dbt Cloud Feature | dbt Core Version Needed |
+| ------------- | -------------- |
+| [Environment variable secret scrubbing](/docs/dbt-cloud/using-dbt-cloud/cloud-environment-variables#handling-secrets)| v0.21.0+ |
+| DAG in the IDE | v0.20.0+|
+| [Metadata API](/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview.md) |v0.19.0+|
+| [Dashboard status tiles](/docs/dbt-cloud/using-dbt-cloud/cloud-dashboard-status-tiles) | v0.19.0+ |
+| [Slim CI](/docs/dbt-cloud/using-dbt-cloud/cloud-enabling-continuous-integration-with-github#slim-ci) | v0.18.0+ |
+
+#### Need help upgrading?
+
+If you want more advice on how to upgrade your dbt projects, check out our [migration guides](docs/guides/migration-guide/upgrading-to-0-20-0) and our Q&A page[to add link].
```

Comment:
This looks really great! My only (somewhat unrelated) comment would be should we call out on the migration guide linked here that we've deprecated 0.20.0 and 0.20.1?

Reply:
We unfortunately don't have a guide for the path versions, so I can only point to 0.20.0